src/views/CheckoutSelectAccount.vue

```diff
@@ -1,20 +1,20 @@
 <template>
     <div class="visible-area">
         <div class="multi-pages" :style="`transform: translate3d(-${(page - 1) * 100}%, 0, 0)`">
-            <LoginSelector @wallet-selected="loginSelected"
+            <LoginSelector @login-selected="loginSelected"
```

Comment:
Surprised to see a change back from login to wallet. Instead of having a Login selector selecting a login ID which is a wallet ID, why not simply having a wallet sector selecting a wallet ID?

Reply:
The LoginSelector comes from the vue components, where it's named like that (stemming from back when Marvin created them). Thus also the login naming.